src/pages/_error.tsx

```
@@ -20,63 +1,16 @@
+function Error({ statusCode }: { statusCode: number }) {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)